### PR TITLE
Fix doc error

### PR DESCRIPTION
**Pull request by sergey144010**
### What bugs did you fix?
Fix doc error
### Optional things below
[] Added your name to the credits
[] Add a update name below:

### CODE REVIEW

**Comment by xXXIMMATTHEWXXx** [comment]
Do you mind making that line have better grammar

**Comment by sergey144010** [comment]
Look now at the end.
https://github.com/SSilence/php-imap-client/blob/master/docs/guide-en/Usage.md
Here is the fixed output code and link.

File: docs/guide-en/Usage.md [hunk 1]
```diff
@@ -125,8 +125,9 @@ We also can save emails
 ```php
 // Note: for slower web servers will less ram use saveEmailSafe()
 $imap->saveEmail('archive/users/johndoe/email_1.eml', 1);
+```
 
-For a full list of methods you can do check docs/Methods.md
+For a full list of methods you can do check part [Methods](docs/guide-en/Methods.md).
```
Comment (xXXIMMATTHEWXXx) [inline docs/guide-en/Usage.md]: This should be "For a full list of methods you can check our [current list of methods](docs/guide-en/Methods.md)."

**Comment by xXXIMMATTHEWXXx** [comment]
Thanks for fixing that btw

**Comment by sergey144010** [comment]
ok, I corrected it.

**Comment by xXXIMMATTHEWXXx** [comment]
Thanks

**Pull request merged**